[2023-04-23 22:59:46,127] INFO _log() L224   * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.35.33:5000 (Press CTRL+C to quit) - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-23 22:59:58,699] INFO _log() L224  127.0.0.1 - - [23/Apr/2023 22:59:58] "[32mGET / HTTP/1.1[0m" 302 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-23 22:59:58,709] INFO _log() L224  127.0.0.1 - - [23/Apr/2023 22:59:58] "[32mGET /openapi HTTP/1.1[0m" 308 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-23 22:59:58,726] INFO _log() L224  127.0.0.1 - - [23/Apr/2023 22:59:58] "GET /openapi/ HTTP/1.1" 200 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-23 22:59:59,115] INFO _log() L224  127.0.0.1 - - [23/Apr/2023 22:59:59] "GET /openapi/static/images/rapidoc.svg HTTP/1.1" 200 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-23 22:59:59,118] INFO _log() L224  127.0.0.1 - - [23/Apr/2023 22:59:59] "GET /openapi/static/images/redoc.svg HTTP/1.1" 200 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-23 22:59:59,124] INFO _log() L224  127.0.0.1 - - [23/Apr/2023 22:59:59] "GET /openapi/static/images/swagger.svg HTTP/1.1" 200 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-23 22:59:59,147] INFO _log() L224  127.0.0.1 - - [23/Apr/2023 22:59:59] "GET /openapi/static/images/apidoc.svg HTTP/1.1" 200 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-23 23:00:00,578] INFO _log() L224  127.0.0.1 - - [23/Apr/2023 23:00:00] "GET /openapi/swagger HTTP/1.1" 200 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-23 23:00:00,601] INFO _log() L224  127.0.0.1 - - [23/Apr/2023 23:00:00] "GET /openapi/static/css/swagger-ui.css HTTP/1.1" 200 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-23 23:00:00,606] INFO _log() L224  127.0.0.1 - - [23/Apr/2023 23:00:00] "GET /openapi/static/js/swagger-ui-bundle.js HTTP/1.1" 200 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-23 23:00:00,608] INFO _log() L224  127.0.0.1 - - [23/Apr/2023 23:00:00] "GET /openapi/static/js/swagger-ui-standalone-preset.js HTTP/1.1" 200 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-23 23:00:00,862] INFO _log() L224  127.0.0.1 - - [23/Apr/2023 23:00:00] "GET /openapi/openapi.json HTTP/1.1" 200 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-23 23:00:01,046] INFO _log() L224  127.0.0.1 - - [23/Apr/2023 23:00:01] "[36mGET /openapi/static/images/swagger.svg HTTP/1.1[0m" 304 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-23 23:00:25,274] INFO _log() L224  127.0.0.1 - - [23/Apr/2023 23:00:25] "GET /openapi/swagger HTTP/1.1" 200 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-23 23:00:25,297] INFO _log() L224  127.0.0.1 - - [23/Apr/2023 23:00:25] "[36mGET /openapi/static/css/swagger-ui.css HTTP/1.1[0m" 304 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-23 23:00:25,305] INFO _log() L224  127.0.0.1 - - [23/Apr/2023 23:00:25] "[36mGET /openapi/static/js/swagger-ui-bundle.js HTTP/1.1[0m" 304 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-23 23:00:25,306] INFO _log() L224  127.0.0.1 - - [23/Apr/2023 23:00:25] "[36mGET /openapi/static/js/swagger-ui-standalone-preset.js HTTP/1.1[0m" 304 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-23 23:00:25,431] INFO _log() L224  127.0.0.1 - - [23/Apr/2023 23:00:25] "GET /openapi/openapi.json HTTP/1.1" 200 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-23 23:15:26,759] INFO _log() L224  127.0.0.1 - - [23/Apr/2023 23:15:26] "GET /produtos HTTP/1.1" 200 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-23 23:28:10,311] INFO _log() L224  127.0.0.1 - - [23/Apr/2023 23:28:10] "POST /produto HTTP/1.1" 200 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-23 23:28:25,368] ERROR log_exception() L1449 Exception on /produto [GET] - call_trace=C:\Python311\Lib\site-packages\flask\app.py L1449
Traceback (most recent call last):
  File "C:\Python311\Lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\flask\app.py", line 1519, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\flask_cors\extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
                                                ^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\flask\app.py", line 1517, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\flask\app.py", line 1503, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\flask_openapi3\scaffold.py", line 109, in wrapper
    resp = _do_wrapper(
           ^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\flask_openapi3\do_wrapper.py", line 135, in _do_wrapper
    response = func(**request_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Projetos\IPT\desenvolvimento-full-stack\desenvolvimento-full-stack-basico\aula-3\meu_app_api\app.py", line 94, in get_produto
    produto_id = query.id
                 ^^^^^^^^
AttributeError: 'ProdutoBuscaSchema' object has no attribute 'id'
[2023-04-23 23:28:25,379] INFO _log() L224  127.0.0.1 - - [23/Apr/2023 23:28:25] "[35m[1mGET /produto?nome=Teste HTTP/1.1[0m" 500 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-23 23:28:29,795] ERROR log_exception() L1449 Exception on /produto [GET] - call_trace=C:\Python311\Lib\site-packages\flask\app.py L1449
Traceback (most recent call last):
  File "C:\Python311\Lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\flask\app.py", line 1519, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\flask_cors\extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
                                                ^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\flask\app.py", line 1517, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\flask\app.py", line 1503, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\flask_openapi3\scaffold.py", line 109, in wrapper
    resp = _do_wrapper(
           ^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\flask_openapi3\do_wrapper.py", line 135, in _do_wrapper
    response = func(**request_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Projetos\IPT\desenvolvimento-full-stack\desenvolvimento-full-stack-basico\aula-3\meu_app_api\app.py", line 94, in get_produto
    produto_id = query.id
                 ^^^^^^^^
AttributeError: 'ProdutoBuscaSchema' object has no attribute 'id'
[2023-04-23 23:28:29,803] INFO _log() L224  127.0.0.1 - - [23/Apr/2023 23:28:29] "[35m[1mGET /produto?nome=Teste HTTP/1.1[0m" 500 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-23 23:28:49,086] INFO _log() L224  127.0.0.1 - - [23/Apr/2023 23:28:49] "GET /produtos HTTP/1.1" 200 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-26 10:30:16,338] INFO _log() L224   * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.35.33:5000 (Press CTRL+C to quit) - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-26 10:30:41,830] INFO _log() L224  127.0.0.1 - - [26/Apr/2023 10:30:41] "[32mGET / HTTP/1.1[0m" 302 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-26 10:30:41,838] INFO _log() L224  127.0.0.1 - - [26/Apr/2023 10:30:41] "[32mGET /openapi HTTP/1.1[0m" 308 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-26 10:30:41,854] INFO _log() L224  127.0.0.1 - - [26/Apr/2023 10:30:41] "GET /openapi/ HTTP/1.1" 200 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-26 10:30:42,253] INFO _log() L224  127.0.0.1 - - [26/Apr/2023 10:30:42] "GET /openapi/static/images/swagger.svg HTTP/1.1" 200 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-04-26 10:30:42,254] INFO _log() L224  127.0.0.1 - - [26/Apr/2023 10:30:42] "GET /openapi/static/images/redoc.svg HTTP/1.1" 200 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
